EhPanda/View/Setting/Login/LoginReducer.swift

```diff
@@ -63,14 +64,16 @@ struct LoginReducer: Reducer {
                 return .none
 
             case .teardown:
-                return .cancel(id: CancelID.self)
+                return .cancel(id: CancelID.teardown)
```

Comment:
It doesn't seem quite right to me.
`CancelID.teardown` is not associated with any operation. 🤔
Maybe we should cancel the `LoginRequest` with `CancelID.login` here, like other teardown actions?
```suggestion
                return .cancel(id: CancelID.login)
```